spec/requests/site_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe SiteController do
```

Comment:
I didn't cover everything in here, but I did focus on the urls that are most likely semi-permanent.